packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx

```diff
@@ -255,6 +269,17 @@ function CodeMirror(props: NoteBodyEditorProps, ref: any) {
 		}
 	}, []);
 
+	const editorPaste = useCallback(() => {
+		const clipboardText = clipboard.readText();
+
+		if (clipboardText) {
+			editorPasteText();
+		} else {
+			// To handle pasting images
+			void onEditorPaste();
+		}
+	}, []);
```

Comment:
Although for now it won't make a difference, it would be best to add the dependencies to the array - so in this case `editorPasteText` and `onEditorPaste`. For now we don't use the eslint rule to check for missing hook dependencies because it has some issues, but if we enable it later on, there will probably be dependencies to fix so it's best to do it as we go. I guess we don't need to add refs to dependencies though.

Reply:
Good point, thanks for the heads up, it's done.